src/components/families/family-form/family-parent-fields/FamilyParentFields.tsx

```diff
@@ -30,6 +32,24 @@ const FamilyParentFields = ({ dynamicFields, family, onChange }: Props) => (
       }
       value={family.parent.last_name}
     />
+    <Field
+      field={{ ...DefaultFields.DATE_OF_BIRTH, role: StudentRole.PARENT }}
+      onChange={(value) => {
+        const dob = value || null;
+        onChange({
+          ...family,
+          parent: {
+            ...family.parent,
+            date_of_birth: dob,
+          },
+        });
+      }}
+      value={
+        family.parent.date_of_birth
+          ? moment(family.parent.date_of_birth).format("YYYY-MM-DD")
+          : ""
+      }
```

Comment:
date_of_birth should already be a string in this format!

```suggestion
      value={family.parent.date_of_birth || ""}
```